<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8" />
  <meta content="width=device-width, initial-scale=1" name="viewport" />
  <meta content="#ffffff" name="theme-color" />
  <meta content="#da532c" name="msapplication-TileColor" />

  
  
  
  
  

  <link href="https://cdnjs.cloudflare.com/ajax/libs/galleria/1.6.1/themes/folio/galleria.folio.min.css" rel="stylesheet" />
  <link href="https://cdn.jsdelivr.net/npm/mapbox-gl@2.1.1/dist/mapbox-gl.min.css" rel="stylesheet" />
  <link href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" rel="stylesheet" />
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <link href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css" rel="stylesheet" />
  <link href="https://snystrom.github.io/deep-thought.css" rel="stylesheet" />

  <title>
    
Endlessly Screaming Into the Void | Purifying monoclonal antibodies (for CUT&amp;RUN or other reasons)

  </title>

  

  
</head>

<body class="has-background-white">
  <nav aria-label="section navigation" class="navbar is-light" role="navigation">
    <div class="container">
      <div class="navbar-brand">
        <a class="navbar-item is-size-5 has-text-weight-bold" href="https:&#x2F;&#x2F;snystrom.github.io">Endlessly Screaming Into the Void</a>
        <a aria-expanded="false" aria-label="menu" class="navbar-burger burger" data-target="navMenu" role="button">
          <span aria-hidden="true"></span>
          <span aria-hidden="true"></span>
          <span aria-hidden="true"></span>
        </a>
      </div>
      <div class="navbar-menu" id="navMenu">
        <div class="navbar-end has-text-centered">
          
          
          
          <a class="navbar-item has-text-weight-semibold" href="https:&#x2F;&#x2F;snystrom.github.io&#x2F;">
            Home
          </a>
          
          <a class="navbar-item has-text-weight-semibold" href="https:&#x2F;&#x2F;snystrom.github.io&#x2F;resume&#x2F;snystrom_web_resume.pdf">
            Resume
          </a>
          
          <a class="navbar-item has-text-weight-semibold" href="https:&#x2F;&#x2F;snystrom.github.io&#x2F;protocols">
            Molecular Biology Protocols
          </a>
          
          
          
          <a class="navbar-item" id="nav-search" title="Search" data-target="#search-modal">
            <span class="icon">
              <i class="fas fa-search"></i>
            </span>
          </a>
          <a class="navbar-item" id="dark-mode" title="Switch to dark theme">
            <span class="icon">
              <i class="fas fa-adjust"></i>
            </span>
          </a>
        </div>
      </div>
    </div>
  </nav>

  
<section class="section">
  <div class="container">
    <div class="columns">
      <div class="column is-8 is-offset-2">
        <article class="box">
          <h1 class="title">
            Purifying monoclonal antibodies (for CUT&amp;RUN or other reasons)
          </h1>
          <p class="subtitle"></p>
          <div class="columns is-multiline is-gapless">
            <div class="column is-8">
              
<span class="icon-text has-text-grey">
  <span class="icon">
    <i class="fas fa-user"></i>
  </span>
  <span>Spencer published on</span>
  <span class="icon">
    <i class="far fa-calendar-alt"></i>
  </span>
  <span><time datetime="2021-09-19">September 19, 2021</time></span>
</span>

            </div>
            <div class="column is-4 has-text-right-desktop">
              
<span class="icon-text has-text-grey">
  <span class="icon">
    <i class="far fa-clock"></i>
  </span>
  <span>6 min,</span>
  <span class="icon">
    <i class="fas fa-pencil-alt"></i>
  </span>
  <span>1050 words</span>
</span>

            </div>
            <div class="column">
              
              
<p>
  Categories:
  
  <a class="has-text-info-dark has-text-weight-semibold" href="https://snystrom.github.io/categories/protocols/">
    <span class="icon-text">
      <span class="icon">
        <i class="fas fa-cube"></i>
      </span>
      <span>Protocols</span>
    </span>
  </a>
  
</p>

              
            </div>
            <div class="column has-text-right-desktop">
              
              
<p>
  Tags:
  
  <a class="has-text-info-dark has-text-weight-semibold" href="https://snystrom.github.io/tags/cut-run/">
    <span class="icon-text">
      <span class="icon">
        <i class="fas fa-tag"></i>
      </span>
      <span>cut&amp;run</span>
    </span>
  </a>
  
</p>

              
            </div>
          </div>
          <div class="content mt-2">
            <p>I spent years of my PhD sequencing mouse instead of <em>Drosophila</em> DNA. Now you don't have to!</p>
<p>Fun fact: if you use monoclonal antibodies from the <a href="https://dshb.biology.uiowa.edu/">Developmental Studies
Hybridoma Bank (DSHB)</a>, they're full of mouse
nuclei &amp; genomic DNA. This has a nasty consequence for low-input assays such as
<a href="https://elifesciences.org/articles/21856">CUT&amp;RUN</a>, since the mouse DNA will
swamp out real signal. This issue is multiplied when using a model organism
with a smaller genome size than mouse (like <em>Drosophila</em>), because outcompeting
the contaminating DNA becomes almost impossible when using a smaller genome.</p>
<p>This issue isn't present during ChIP-seq because typically, antibodies are
first bound to magnetic beads, which are washed &amp; purified away from the dirty
supernatant before IP-ing chromatin on the beads. This approach doesn't work
for CUT&amp;RUN because it's <em>in-situ</em>, the antibody must be added in solution to
the tissue. Hence, we need a different way to purify the antibody before the
digest that retains activity while removing contaminating DNA. Of course,
there are other reasons you may not want DNA or nuclei in your antibody, this
protocol should work for those, too. </p>
<h2 id="is-my-antibody-contaminated">Is my antibody contaminated?</h2>
<p>Great question. Here are a few easy ways to check:</p>
<ol>
<li>Quantify nucleic acid concentration of your raw antibody using a Qubit. 
<ul>
<li>I like the high-sensitivity dsDNA Qubit kit for this since it is very specific for DNA</li>
</ul>
</li>
<li>DAPI stain a small volume of antibody &amp; mount on a hemocytometer
<ul>
<li>If your sample is anything like mine, it will look like the night sky, full of in-tact and blebbing mouse nuclei</li>
</ul>
</li>
</ol>
<p>You can repeat these steps after purification, and if it worked you should read 0ng/uL of DNA, and see 0 nuclei. Yes it's that clean.</p>
<h2 id="how-do-i-fix-it">How do I fix it?</h2>
<p>After trying a lot of things that didn't work, Kami Ahmad suggested I try using
magentic proteinG beads to purify the antibody away from the nuclei/DNA crap.
It worked great. </p>
<p>The protocol works as follows:</p>
<ol>
<li>Bind the antibody to magnetic beads</li>
<li>Wash away the supernatant contaminated with nuclei &amp; DNA</li>
<li>Elute the antibody from the beads using an acidic pH (2.5 for proteinG beads, ~3.0-3.3 for proteinA beads)</li>
<li>Quickly neutralizing the acidic elution buffer to prevent denaturing the antibody</li>
</ol>
<p>Honestly the hardest part is making the buffers, and they're really easy to make.</p>
<h3 id="some-thoughts">Some thoughts</h3>
<p>I let the antibody incubate for 40 minutes on beads, even though the thermo
manual for dynabeads says you can do it in 10. This was mostly because I had an
older set of beads and I was so tired of things not working by this point in
grad school that I didn't mind waiting an extra 1/2 hour. </p>
<p>I do 2 rounds of elution to maximize recovery of antibody &amp; to cut down on the
time the antibody sits in elution buffer. I always pooled the 2 eluates and
never tested whether the second one was high yield. </p>
<p>Another great test after purification is to perform immunofluorescence using
the purified and unpurified antibodies in parallel. If the purification worked
correctly, you should observe similar intensity staining using pre- vs
post-purification antibody. To correctly account for the difference in antibody
volumes, treat the purified antibody as a dilution of the stock. For exmple, if
you input 12uL unpurified antibody, after purification you are left with 120uL of purified antibody. That's a 1:10 dilution.
So if in a normal immunofluorescence experiment you would use a 1:100 dilution
of stock antibody, you would use a 1:10 dilution of the purified antibody. Use
this same approach when determining the concentration of purified antibody to
use for CUT&amp;RUN.</p>
<p>I use a Tris-Glycine elution strategy, but there are other <a href="https://www.thermofisher.com/order/catalog/product/21027#/21027">gentle antibody
elution
buffers</a> which
may be better for certain antibodies. If you use these, you also have to use an
ion exchange column (like a <a href="https://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-purification-isolation/protein-dialysis-desalting-concentration/zeba-desalting-products/zeba-spin-desalting-columns.html">Zeba
column</a>
to swap the buffers because they're full of chaotropic agents (I think, I dunno
what's in them, although I tried to find out... see references). This protocol
is so easy to do, I'd try the Tris-Glycine approach first, then if the antibody
loses activity in immunofluorescence, you can try a different buffer. But first
make sure you're not letting the elution sit too long! Try shortening the
elution time and see if that helps first.</p>
<p>Good luck!</p>
<h2>
<a href="/protocol_pdfs/mouse_antibody_purification_using_pGDynabeads_v1.pdf" target="_blank">
<span class="icon-text">
	<span class="icon">
		<i class="fas fa-download"></i>
	</span>
	&nbsp;&nbsp;Protocol download here
</span>
</a>
</h2>
<br>
</br>
<h3 id="references">References:</h3>
<ul>
<li>
<p><a href="https://www.neb.com/protocols/0001/01/01/purification-of-igg-using-protein-ag-magnetic-beads">NEB Antibody Purification w/ pAG beads</a></p>
</li>
<li>
<p><a href="http://cshprotocols.cshlp.org/content/2006/1/pdb.rec8303.full">CSH Phosphate Buffer Recipe</a></p>
</li>
<li>
<p><a href="https://www.bio-rad-antibodies.com/static/2015/proteus/protein-g-handbook.pdf">protein G manual</a></p>
<ul>
<li>BioRad protein G purification manual (This one is GREAT)</li>
</ul>
</li>
<li>
<p><a href="https://www.thermofisher.com/order/catalog/product/21027#/21027">Pierce Gentle ab elution buffer</a></p>
</li>
<li>
<p><a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0011176_Gentle_AgAb_Bind_Elution_Buff_UG.pdf">Sigma gentle pAG buffers</a></p>
</li>
</ul>
<ul>
<li>Thought: using phosphate-free system could be better because of CUT&amp;RUN in addition to being better for abs
<ul>
<li>Note: This wasn't an issue for me, but could be in some cases.</li>
</ul>
</li>
</ul>
<ul>
<li>
<p><a href="https://dshb.biology.uiowa.edu/product-forms">DSHB Product Forms</a></p>
<ul>
<li>Antibodies are delivered in PBS</li>
<li>concentrate is supernatant run through 100kda size exclusion filter</li>
</ul>
</li>
<li>
<p><a href="https://www.jbc.org/content/243/3/651.full.pdf">Paper on MNase inhibitors/biochemistry</a></p>
<ul>
<li>dNdp's inhibit MNase activity (dA/Tp in particular)</li>
<li>ie di-phosphate nucleotides w/ 3' P.</li>
</ul>
</li>
<li>
<p><a href="https://www.gbiosciences.com/image/pdfs/protocol/Gentle_Elution_Buffers.pdf">G biosci gentle elution buffer</a></p>
<ul>
<li>use Tris/HEPES/MOPS ph 7.2-7.4 buffer w/ 150mM salt (no phosphates)</li>
<li>Neutralization Buffer: 1M Tris, pH8.0</li>
</ul>
</li>
<li>
<p>Sadly, I dug up an amazing set of 2 comprehensive proteinA and proteinG antibody purification manuals from thermofisher (or maybe pierce) that were of extreme help that I can no longer dig up. They were like 40 pages each with detailed troubleshooting &amp; biochemistry sections. If anyone finds something like this, please reach out. I asked thermo and they couldn't find them either. Maybe I'm crazy.</p>
</li>
</ul>

          </div>
        </article>
      </div>
      
    </div>
  </div>
</section>


  
  <section class="modal" id="search-modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">Search</p>
      </header>
      <section class="modal-card-body">
        <div class="field mb-2">
          <div class="control">
            <input class="input" id="search" placeholder="Search this website." type="search" />
          </div>
        </div>
        <div class="search-results">
          <div class="search-results__items"></div>
        </div>
      </section>
    </div>
    <button aria-label="close" class="modal-close is-large"></button>
  </section>
  


  



  



  <footer class="footer py-4">
    <div class="content has-text-centered">
      <p>
				Aaaaaaaaaahhhhhhhhhh!!! > /dev/null
      </p>
    </div>
  </footer>

  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/galleria/1.6.1/galleria.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/mermaid@8.9.2/dist/mermaid.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/chart.xkcd@1.1.13/dist/chart.xkcd.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/mapbox-gl@2.1.1/dist/mapbox-gl.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/galleria/1.6.1/themes/folio/galleria.folio.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/elasticlunr/0.9.6/elasticlunr.min.js"></script>
  <script src="https://snystrom.github.io/search_index.en.js"></script>
  <script src="https://snystrom.github.io/js/site.js"></script>

  




</body>

</html>
